CHANGELOG.md

```diff
@@ -55,6 +55,10 @@
   [Ben P](https://github.com/ben-p-commits)
   [#5263](https://github.com/realm/SwiftLint/issues/5263)
 
+* Make `sorted_enum_cases` rule's comparison case-insensitive to avoid unexpected ordering 
```

Comment:
```suggestion
* Make `sorted_enum_cases` rule's comparison case-insensitive to
  avoid unexpected ordering.  
```

Please also append two spaces to the end of the description. They are required to enforce a line break in the rendered Markdown.